src/dotnet/commands/RestoringCommand.cs

```diff
@@ -20,7 +20,8 @@ private IEnumerable<string> ArgsToForwardToRestore()
                 !a.StartsWith("/t:") &&
                 !a.StartsWith("/target:") &&
                 !a.StartsWith("/ConsoleLoggerParameters:") &&
-                !a.StartsWith("/clp:"));
+                !a.StartsWith("/clp:") &&
+                !a.StartsWith("/p:TargetFramework"));
```

Comment:
@nguerrera I have both because we have different commands passing this down in different ways. I could have gone the route of changing the commands, but this was easier then.